packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -127,6 +128,20 @@ export class ProjectService {
             user.organizationUuid,
             data,
         );
+        analytics.track({
```

Comment:
@owlas is `create` from line 148 also executed after the preview (if so, this project.created is redudant and we will be tracking the same event twice)

Reply:
nope it's a separate endpoint!

Reply:
merge away if you're happy